apps/web/src/pages/Pools/AssessmentPlanBuilderPage/AssessmentPlanBuilderPage.tsx

```diff
@@ -16,6 +16,8 @@ import {
   Pending,
   Separator,
 } from "@gc-digital-talent/ui";
+import Sidebar from "@gc-digital-talent/ui/src/components/Sidebar";
```

Comment:
If you add `Sidebar` to the `ui` barrel export file we can add this to the import statement above.

Reply:
Oh yeah, I like that.
[add sidebar to barrel export](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8290/commits/98d648d1ef096773507435cd63f739fc11906bbe)